CHANGES/1516.bugfix

```diff
@@ -0,0 +1 @@
+Fixed version ordering in CollectionVersionSearchFilter
```

Comment:
```suggestion
Fixed version ordering in ``CollectionVersionSearchFilter``.
```
But maybe you want to reformulate this in user terms. Which filter on which endpoint was broken? Users do not know about our Python classes.